.github/workflows/push-docker-images-release.yml

```diff
@@ -17,17 +17,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+    # See https://github.com/google-github-actions/auth#authenticating-to-container-registry-and-artifact-registry
     - id: "gcp-auth"
       name: "Authenticate to GCP"
       uses: "google-github-actions/auth@v0"
       with:
-        workload_identity_provider: ${{ secrets.GCP_ARTIFACT_PUBISHER_WORKLOAD_IDENTITY_PROVIDER }}
-        service_account: ${{ secrets.GCP_ARTIFACT_PUBLISHER_DEPLOY_SERVICE_ACCOUNT }}
+        workload_identity_provider: ${{ secrets.GCP_ARTIFACT_READER_WORKFLOW_IDENTITY_PROVIDER }}
+        service_account: ${{ secrets.GCP_ARTIFACT_READER_SERVICE_ACCOUNT }}
```

Comment:
FYI, this change was applied to the wrong file (ci-build.yml should be using the READER account, not this)